sqlmesh/core/snapshot/evaluator.py

```diff
@@ -390,12 +390,25 @@ def _create_snapshot(
             is_dev=is_dev,
         )
 
+        evaluation_strategy = _evaluation_strategy(snapshot, self.adapter)
+
         with self.adapter.transaction(TransactionType.DDL), self.adapter.session():
             self.adapter.execute(snapshot.model.render_pre_statements(**render_kwargs))
 
-            _evaluation_strategy(snapshot, self.adapter).create(
-                snapshot.model, table_name, **render_kwargs
-            )
+            if is_dev and snapshot.is_materialized and self.adapter.SUPPORTS_CLONING:
+                tmp_table_name = f"{target_table_name}__schema_migration_source"
```

Comment:
Why does it have a suffix of `schema_migration_source`?

Reply:
Because we use this table as a source of schema when altering the clone. Would you suggest a different suffix?

Reply:
Got it now. Thanks!